src/Raven.Studio/typescript/viewmodels/database/indexes/indexPerformance.ts

```diff
@@ -1222,7 +1222,8 @@ class indexPerformance extends viewModelBase {
                     countsDetails += `<div class="tooltip-li">Average Document Size: <div class="value">${generalUtils.formatBytesToSize(parentStats.DocumentsSize.SizeInBytes / parentStats.InputCount)}</div></div>`;
                 }
 
-                countsDetails += `<div class="tooltip-li">Managed Allocation Size: <div class="value">${parentStats.AllocatedBytes.HumaneSize}</div></div>`;
+                countsDetails += `<div class="tooltip-li">Managed Allocation Size (accumulated): <div class="value">${parentStats.AllocatedManagedBytes.HumaneSize}</div></div>`;
```

Comment:
Maybe use `Managed Allocations (accumulated):` here? It's shorter

Reply:
Done